### PR TITLE
Revise serialization docs per revised guidance on adding copilot highlights to docs

### PR DESCRIPTION
**Pull request by anandmeg**
---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/standard/serialization/system-text-json/customize-properties.md](https://github.com/dotnet/docs/blob/9753810f6e570c6a56a60d9995e0d159df614650/docs/standard/serialization/system-text-json/customize-properties.md) | [How to customize property names and values with System.Text.Json](https://review.learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/customize-properties?branch=pr-en-us-43997) |
| [docs/standard/serialization/system-text-json/deserialization.md](https://github.com/dotnet/docs/blob/9753810f6e570c6a56a60d9995e0d159df614650/docs/standard/serialization/system-text-json/deserialization.md) | [How to read JSON as .NET objects (deserialize)](https://review.learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/deserialization?branch=pr-en-us-43997) |
| [docs/standard/serialization/system-text-json/how-to.md](https://github.com/dotnet/docs/blob/9753810f6e570c6a56a60d9995e0d159df614650/docs/standard/serialization/system-text-json/how-to.md) | ["How to serialize JSON in C#"](https://review.learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/how-to?branch=pr-en-us-43997) |
| [docs/standard/serialization/system-text-json/migrate-from-newtonsoft.md](https://github.com/dotnet/docs/blob/9753810f6e570c6a56a60d9995e0d159df614650/docs/standard/serialization/system-text-json/migrate-from-newtonsoft.md) | [Migrate from Newtonsoft.Json to System.Text.Json](https://review.learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/migrate-from-newtonsoft?branch=pr-en-us-43997) |

### CODE REVIEW

File: docs/standard/serialization/system-text-json/customize-properties.md [hunk 1]
```diff
@@ -271,83 +266,19 @@ The following example shows you how to use Copilot to modify existing code to cu
    ```
 
 1. In Copilot Chat, enter the following prompt to modify the code to customize names and order of the JSON serialization output.
+   
+    ```copilot-prompt
+   `#Example.cs` modify code to use `System.Text.Json` to customize property names and order of JSON output from serialization.
```
Comment (gewarren) [inline docs/standard/serialization/system-text-json/customize-properties.md:271]: Why do we need backticks for the file name and namespace? The entire prompt is already code-fenced so not needed for localization.

File: docs/standard/serialization/system-text-json/deserialization.md [hunk 2]
```diff
@@ -90,63 +90,17 @@ To deserialize from UTF-8, call a <xref:System.Text.Json.JsonSerializer.Deserial
...
-}
-
-**Example Output**
-FirstName: John
-LastName: Doe
-Age: 30
-
-This code defines a Person class with properties that match the JSON string. It then uses `JsonSerializer.Deserialize` to convert the JSON string into a `Person` object and prints the properties to the console.
+```copilot-prompt
+Generate code to use `System.Text.Json` to deserialize a JSON string {"FirstName":"John","LastName":"Doe","Age":30} to an equivalent .NET object.
```
Comment (gewarren) [inline docs/standard/serialization/system-text-json/deserialization.md:98]: Same question here.

File: docs/standard/serialization/system-text-json/how-to.md [hunk 3]
```diff
@@ -111,67 +111,18 @@ Starting in .NET 9, you can also customize the indent character and size using <
...
-        Console.WriteLine(jsonString);
-    }
-}
-
-**Example Output**
-{"FirstName":"John","LastName":"Doe","Age":30}
-
-This code defines a `Person` class, creates an instance of it, and then serializes it to a JSON string using `JsonSerializer.Serialize`. The resulting JSON string is then printed to the console.
+```copilot-prompt
+Generate code to use `System.Text.Json` to serialize an object to a JSON string.
```
Comment (gewarren) [inline docs/standard/serialization/system-text-json/how-to.md:119]: Same question.

**Review by gewarren — COMMENTED** [review]
Thanks!

File: docs/standard/serialization/system-text-json/customize-properties.md (hunk 1 above)
Comment (anandmeg) [reply docs/standard/serialization/system-text-json/customize-properties.md]: leftover from when it was text

**Review by gewarren — APPROVED** [review]

**Review by gewarren — COMMENTED** [review]
Lint

**Pull request merged**